packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3532,8 +3533,7 @@ export default class Meeting extends StatelessWebexPlugin {
           return Promise.reject(error);
         });
     }
-
-    return undefined;
+    LoggerProxy.logger.error('Meeting:index#beRightBack --> Not a multistream meeting');
```

Comment:
This would trigger if there if `this.mediaProperties.webrtcMediaConnection` is not defined, so I think we need to tweak this error message a bit or maybe have 2 separate error messages.